code/datums/quirks/negative.dm

```diff
@@ -6,6 +6,188 @@
 #define LOCATION_BACKPACK "in your backpack"
 #define LOCATION_HANDS "in your hands"
 
+// Red damage quirks start
+/datum/quirk/minor_red
+	name = "Minor red damage weakness"
+	desc = "You take 10% more RED damage"
+	value = -1
+	gain_text = "<span class='danger'>You feel a bit weaker to RED damage</span>"
+	medical_record_text = "This patient is observed to take 10% more physical damage"
```

Comment:
The medical records sound flavorless. I know there's a slim chance people are actually going to see them but saying "Patient has lower pain tolerance" or "Patient's mental instability is dangerously high" sounds more immersive than "takes 10% more red" or "takes double white."